python/lsst/ts/MTAlignment/alignmentModel.py

```diff
@@ -0,0 +1,489 @@
+
+import asyncio
+from enum import IntEnum
+
+
+class LaserStatus(IntEnum):
+    LASERNOTCONNECTED = -1
+    LASEROFF = 0
+    LASERON = 1
+
+
+class TrackerStatus(IntEnum):
+    READY = 1
+    TWOFACE = 2
+    ADM = 3
+    DRIFT = 4
+    EMP = 5
+    ERR = 6
+
+
+class AlignmentModel():
+
+    def __init__(self):
+        self.server_address = ("172.17.0.3", 50000)
```

Comment:
Hard-coded IP/Port. Can you convert these to parameters? A tuple is supposed to be unchanged it is better to have `self.host` and `self.port`